go/store/prolly/message/prolly_map.go

```diff
@@ -63,6 +65,10 @@ func (s ProllyMapSerializer) Serialize(keys, values [][]byte, subtrees []uint64,
 		valTups = writeItemBytes(b, values, valSz)
 		serial.ProllyTreeNodeStartValueOffsetsVector(b, len(values)-1)
 		valOffs = writeItemOffsets(b, values, valSz)
+		if len(s.ValDesc.Addrs) > 0 {
+			serial.ProllyTreeNodeStartValueAddressOffsetsVector(b, len(values)*len(s.ValDesc.Addrs))
```

Comment:
`len(values)*len(s.ValDesc.Addrs)` is only accurate if every blob field is non-null

Reply:
isn't it the same issue with other null fields? or do you do null counting somewhere